Components/SearchController.cs

```diff
@@ -155,11 +121,9 @@ private List<SearchResult> FuzzySearchOpenWindows(List<Window> openWindows)
                     var titleMatch = FuzzyMatching.FindBestFuzzyMatch(window.Title, searchString.SearchText);
                     var processMatch = FuzzyMatching.FindBestFuzzyMatch(window.ProcessName, searchString.SearchText);
 
-                    if ((titleMatch.Count != 0 || processMatch.Count != 0) &&
-                                window.Title.Length != 0)
+                    if (titleMatch.Any() || processMatch.Any() && window.Title.Any())
```

Comment:
this is not the same condition as the original, is this intended?

Reply:
Well it should be, I just remove Count!=0 to Any to make it more clear, which should be the same.

Reply:
The original says either titleMatch.Count != 0 or processMatch.Count != 0 as one condition and window.Title.Length != 0. 

New change says titleMatch.Any() or otherwise processMatch.Any() and window.Title.Any() as one condition.

Difference lies in the bracket

Reply:
Oh sorry, you are right, I will change that.